# config_pyiot.yaml 
# IoT Configuration
config_pyiot:
  mqtt:
    server_url: "broker.mqttdashboard.com"        # http://www.hivemq.com/demos/websocket-client/
    # server_url: "test.mosquitto.org"			  # Alternative to HiveMQ
    server_port: 1883
    topic_sub: "eyrc/kanishka/iot_to_ros"          # <unique_id> = xYzqLm
    topic_pub: "eyrc/kanishka/ros_to_iot"          # <unique_id> = xYzqLm
    qos: 0

    sub_cb_ros_topic: "/ros_iot_bridge/mqtt/sub"   # ROS nodes can listen to this topic to receive data from MQTT
    
  google_apps:
    spread_sheet_id: "1gmrC1sCzUAQ_NO2QHd9YSQvl0fSJT-nAUC2LL2ZwPj8"

